roles/ffh.mesh_announce/templates/ferm.conf.j2

```diff
@@ -1,7 +1,10 @@
 domain (ip ip6) {
   table filter {
     chain MESH_INPUT {
-      proto udp dport (1001) ACCEPT;
+      proto udp {
+        dport (1001) ACCEPT;
+        saddr fe80::/64 sport 1001 dport 32768:61000 mod comment comment "mesh_respondd_reply" ACCEPT;
```

Comment:
Can we remove `dport` here?

Reply:
Why is the line necessary at all? This should be handled by conntrack anyways, shouldn't it?

Reply:
> Can we remove dport here?

Yes, we could. But this is more specific.

> This should be handled by conntrack anyways, shouldn't it?

I can assure it is not optional. I'm not sure, whether conntrack is clever enough for this; effectively it's an 'unrelated' unicast answer to a multicast query.

Reply:
Have you seen the comment above?
https://github.com/freifunkh/ansible/pull/197#issuecomment-801935031